pkgs/by-name/go/google-play/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "google-play";
+  version = "2024-10-18";
```

Comment:
Why not package https://github.com/3052/google/releases/tag/v1.5.8 ?

Reply:
Because it does not have releases, I didn't notice that it has tags. Will change to use tags in a bit.